build/ci/release/Jenkinsfile

```diff
@@ -47,7 +47,8 @@ pipeline {
         }
         unsuccessful {
             script {
-                def msg = "Release job failed!\r\n" + env.BUILD_URL
+                def msg = "Release job failed! \r\n" +
+                          "env.BUILD_URL"
```

Comment:
doesn't this need to be `${BUILD_URL}` similar to other vars?

Reply:
Good catch 😃 
Fixed